consensus/src/network_interface.rs

```diff
@@ -106,7 +106,7 @@ pub fn network_endpoint_config() -> AppConfig {
     AppConfig::p2p(
         protos,
         aptos_channel::Config::new(NETWORK_CHANNEL_SIZE)
-            .queue_style(QueueStyle::LIFO)
```

Comment:
Was this done intentionally or was just an oversight?

Reply:
it's too old, I don't even remember this piece of code. it's not expected to me, we have the LIFO queue here for protection https://github.com/aptos-labs/aptos-core/blob/main/consensus/src/network.rs#L254